pkg/apis/certmanager/v1/types_certificate.go

```diff
@@ -231,6 +231,10 @@ type CertificatePrivateKey struct {
 	// No other values are allowed.
 	// +optional
 	Size int `json:"size,omitempty"` // Validated by webhook. Be mindful of adding OpenAPI validation- see https://github.com/jetstack/cert-manager/issues/3644
+
+	// Other formats in which the private key should be written to the secret
+	// +optional
+	AdditionalKeyFormats []AdditionalKeyFormat `json:"additionalKeyFormats,omitempty"`
```

Comment:
I think this should be named `additionalOutputFormats` and set at the root of the `Spec` struct (e.g. `spec.additionalOutputFormats`).

Otherwise I don't think it makes sense for us to support e.g. `CombinedPEM` here in future, as the positioning of it _today_ implies that it's only an additional format for the *privateKey*, which isn't true with CombinedPEM

Reply:
As discussed on slack we agreed to use the `additionalOutputFormats` in the `CertificateSpec` .